src/components/App.tsx

```diff
@@ -85,19 +89,19 @@ const App = () => {
               id="lists-wrapper"
               className="flex px-6 pt-3 lists-wrapper overflow-x-auto"
               ref={provided.innerRef}
-              {...[provided.droppableProps]}
+              {...provided.droppableProps}
```

Comment:
For the longest time, there was a console error stating "Invalid attribute named 0". This was the cause. See https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/api/droppable.md